src/LetterChanger.java

```diff
@@ -1,57 +1,52 @@
+import java.util.List;
 import java.util.Scanner;
 
 class LetterChanger {
     private static Scanner scanner = new Scanner(System.in);
-
-    public static void main(String[] args) {
-        int numberOfWords = getNumberOfWords();
-        String[] words = readWords(numberOfWords);
-        String change = readChangeString();
-        if (change == "upper") {
-            changeWordsToUpperCase(words);
-        } else if(change == "lower") {
-            changeWordsToLowerCase(words);
-        } else {
-            System.out.println("Nieprawidłowy tekst, zostawiam oryginalny tekst");
+    OperationsOnWords onWords = new OperationsOnWords();
+
+    private final String commendToUpperCase = "upper";
```

Comment:
command*

Reply:
Also if its final change the naming to be appropriate. for example COMMAND_TO_UPPERCASE